server.go

```diff
@@ -84,3 +87,18 @@ func main() {
 		goji.Serve()
 	}
 }
+
+func PanicRecover(c *web.C, h http.Handler) http.Handler {
```

Comment:
__[golint]__
* [golint] exported function PanicRecover should have comment or be unexported